packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -2729,18 +2729,23 @@ class RenderFittedBox extends RenderProxyBox {
   void performLayout() {
     if (child != null) {
       child!.layout(const BoxConstraints(), parentUsesSize: true);
-      switch (fit) {
-        case BoxFit.scaleDown:
-          final BoxConstraints sizeConstraints = constraints.loosen();
-          final Size unconstrainedSize = sizeConstraints.constrainSizeAndAttemptToPreserveAspectRatio(child!.size);
-          size = constraints.constrain(unconstrainedSize);
-        case BoxFit.contain:
-        case BoxFit.cover:
-        case BoxFit.fill:
-        case BoxFit.fitHeight:
-        case BoxFit.fitWidth:
-        case BoxFit.none:
-          size = constraints.constrainSizeAndAttemptToPreserveAspectRatio(child!.size);
+      if (child!.size.isEmpty) {
+        size = constraints.biggest;
```

Comment:
Our default is usually `constraints.smallest` for these cases (see also the else branch bellow). Was there a good reason to use biggest here? Or should this also be smallest?

Reply:
My bad, I made some mistake in previous testing and thought the default behaviour is `constraints.smallest`

Reply:
I've changed it to smallest. And I updated `computeDryLayout` to match `performLayout` since I forgot about `computeDryLayout` perviously.